Cargo.toml

```diff
@@ -75,7 +75,7 @@ tracing = { version = "0.1", default-features = false }
 tracing-attributes = "0.1"
 url = "2.5"
 warp_lambda = "0.1.4"
-wsts = "9.2.0"
+wsts = "11.0.0"
```

Comment:
What's the purpose of this one being `11.0.0` and the `signer` once being a git commit hash?

Reply:
I was testing wsts main branch to make sure it was ready for tag and publish.  I just checked in a change that moves the top level dependency to `12.0.0` and the signer dependency to `workspace`.

Reply:
This branch is a fork of the `update-wsts-11` branch which is why the top-level dependency was `11.0.0`

Reply:
Ah okay, thx 👍